### DISCUSSION TITLE
Welcome to bhtom2 Discussions!

### DISCUSSION THREAD

**Discussion by maja-jablonska** [BHTOM-Team/bhtom2#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wyrzykow** [comment]
When non-Gaia target is created, Gaia Alerts broker should check if by any chance there is no Gaia Alert name there.

**Comment by wyrzykow** [comment]
Each Broker should have also a field, returning an automated Acknowledgment text, e.g. MACHO, https://macho.nci.org.au/, has this text: "This paper utilizes public domain data obtained by the MACHO Project, jointly funded by the US Department of Energy through the University of California, Lawrence Livermore National Laboratory under contract No. W-7405-Eng-48, by the National Science Foundation through the Center for Particle Astrophysics of the University of California under cooperative agreement AST-8809616, and by the Mount Stromlo and Siding Spring Observatory, part of the Australian National University."

**Reply by wyrzykow** [reply]
and method in Publication returning all Acknowledgments for all brokers, which return data used in a given light curve.
